archimedes/src/app/app.component.html

```diff
@@ -37,6 +37,13 @@
             </label>
         </div>
     </div>
+    <div class="row" *ngIf="(selected.program || selected.news || selected.image) && query && query.length < 3">
```

Comment:
We could add `mt-4` here to match the spacing that the row that contains results has

Reply:
I thought since it was more closely related to the search text box it should have less margin above it. Now I am thinking maybe it could be moved above the checkboxes with less padding around the alert text. Thoughts?